src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -103,7 +103,7 @@ public abstract partial class Compilation
             return set;
         }
 
-        internal abstract AnalyzerDriver AnalyzerForLanguage(ImmutableArray<DiagnosticAnalyzer> analyzers, AnalyzerManager analyzerManager);
+        internal abstract AnalyzerDriver AnalyzerForLanguage(ImmutableArray<DiagnosticAnalyzer> analyzers, DefaultCompilationWithAnalyzersHost analyzerManager);
```

Comment:
>AnalyzerForLanguage [](start = 41, length = 19)

I realize this is not your API, but why is this method not a verb?  Should be called GetAnalzyerDriver or CreateAnalyzerDriver